client/src/modules/chapters/graphql/mutations.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from '@apollo/client';
+
+export const joinChapter = gql`
+  mutation joinChapter($chapterId: Int!) {
+    joinChapter(chapterId: $chapterId) {
+      user {
+        name
+      }
+      chapter_role {
+        name
+      }
+      subscribed
```

Comment:
Do we need all this?  I think it's better to keep the returns minimal and only return what we absolutely have to.